### DISCUSSION TITLE
[Suggestion] canvas cursor shape and size

### DISCUSSION THREAD

**Discussion by liganega** [pubpub-zz/ppInk#194] [General]
ppink is great, and I love to use it.
But there is one point that bothers me.
It's about Canvas cursor.

Currently there are two options for Canvas cursor: Pen tip or Arrow
But each option could be better.

1) Pen tip: the pen tip is two small and hard to find sometimes. With the mouse wheel, one could make it bigger, but then the font size becomes bigger together. 
Both factors should be independent from each other.

2) Arrow: It is just too big and hide the main texts or iamges. It would be nice if one could make it transparent or use other images such as (thin) pencil-like images.

**Comment by pubpub-zz** [comment]
@liganega,
the arrow cursor can customize at will. modify the cursorarrow.ico file drawing a small arrow (or anything else). you should leave transparency around to be sure to prevent resizing. You also replace it with a png file(same name)
If you found a good file, feel free to publish it in this thread 😊

**Reply by pubpub-zz** [reply]
some inputs are also available in this thread #146

**Reply by liganega** [reply]
Thank you very much for not forgetting my question.

However, how about put an option to choose the shape and size of the arrow cursor directly in the app? ^^:)

**Reply by pubpub-zz** [reply]
Although it was easy the remame the defaults files, I've added some options in config.ini (I will not put them in the options dialogbox) to modify the name and the size.
[ppInk (2).zip](https://github.com/pubpub-zz/ppInk/files/11070120/ppInk.2.zip)
If you want to try
